src/couch_mrview_http.erl

```diff
@@ -424,6 +424,16 @@ row_to_json(Id0, Row) ->
     ?JSON_ENCODE(Obj).
 
 
+check_compatible(K1, K2, Props) ->
+    case {lists:keymember(K1, 1, Props), lists:keymember(K2, 1, Props)} of
+        {true, true} ->
+            Err = ["`", K1, "` and `", K2,"` are incompatible"],
```

Comment:
@eiri, agreed. io_lib:format would be better and more consistent with other error messages in that code. Will update the pr.